app/views/podcast_planner/_form_side.html.erb

```diff
@@ -0,0 +1,45 @@
+<div class="card">
+  <div class="card-header-primary">
+    <h5 class="card-title"><%= t('.header.preview') %></h5>
+  </div>
+  <%= turbo_frame_tag "preview", target: "_top" do %>
+    <% if @planner.dates.present? %>
+      <%= form_with(
+            url: podcast_planner_path(@podcast),
+            method: :put
+          ) do |form| %>
+          <div class="card-body">
+            <p><%= "Publish Time: #{l(@planner.publish_time, format: :time_12_hour)}" %></p>
+            <p><%= "Number of Segments: #{@planner.segment_count}" %></p>
+            <p><%= "Number of Drafts: #{@planner.dates.count}" %></p>
+            <ul class="list-group list-group-numbered">
+              <% @planner.dates.each do |date| %>
+                <li class="list-group-item"><%= l(date, format: :day_and_date) %></li>
+              <% end %>
+            </ul>
+          </div>
+          <div class="card-footer">
+            <%= form.hidden_field :week_condition, value: @planner.week_condition %>
+            <%= form.hidden_field :period, value: @planner.period %>
+            <%= form.hidden_field :start_date, value: @planner.start_date %>
```

Comment:
Why is it necessary to pass all the how-to-generate-the-calendar params again?  You already have the array of timestamps right?

I'd think it would be sufficient to pass just the segment_count and the array of UTC timestamps. Which would also let the user edit those dates before submitting (Publish actually lets you select/unselect arbitrary days on the calendar - and I think eventually we'll want that here).